health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/UploadDataMapping.js

```
@@ -348,6 +390,222 @@ function UploadDataMapping({ formData, onSelect, currentCategories }) {
+    }
+
+    // Capacity conversion
+    const capacityKey = t(Schemas?.find((i) => i.description === "Capacity")?.name);
+    if (data[capacityKey] !== undefined) {
+        data[capacityKey] = Number(data[capacityKey]);
+    }
+    const ajv = new Ajv({ strict: false }); // Initialize Ajv
+    let validate = ajv.compile(translatedSchema[type]);
+    const errors = []; // Array to hold validation errors
+    
+    data = [data]
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)